docs/development_testing.md

```diff
@@ -68,6 +68,12 @@ You can also run linting tests against all changed files, as reported by `git di
 script/lint
 ```
 
+In case you want to check the code coverage for your new component, run the following from the root of the repository:
+
+````shell
+pytest ./tests/components/<your_component>/ --cov=./homeassistant/components/<your_component>/ --cov-report term-missing -vv
```

Comment:
The argument after `--cov` should be the import path with dots instead of slashes.

Reply:
Thanks, wasn't aware of that. It's fixed.